### ISSUE TITLE
Google Analytics and Matomo not being added if GTM is not added

### ISSUE THREAD

**Issue by MrBuchnev** [sulu/sulu#6135]
| Q | A
| --- | ---
| Sulu Version | 2.2.0
| PHP Version | 7.2
| DB Version | MySQL 5.7.32
| Browser Version | N/A

# Actual Behavior

When trying to add Google Analytics and/or Matomo via webspaces panel, they are not added to the website. Either one of them (or both) only appear after Google Tag Manager is added.

# Expected Behavior

Each analytics tool should be added on its own, without having to add GTM.

# Steps to Reproduce

1) Go to Analytics tab in the Webspaces panel;
2) Add GA and/or Matomo to all domains;
3) Remove cache (optional);
4) Reload website and go to Inspector.

For either GA or Matomo to appear, try adding Google Tag Manager right after step 1.

**Comment by Prokyonn** [comment]
I just tested this out with a new Sulu-Skeleton 2.2.11 project and it seems to work without any problems.

Have you maybe forgotten to activate it for either your domain or all domains?

**Comment by MrBuchnev** [comment]
@Prokyonn , I'm always selecting all domains (which is a single one in my case), and yet the problem persists.

The problem first became apparent on an already functioning project, so I decided to create a clean new Sulu project and test the bug there. Unfortunately, the problem persists on a clean one for me as well.

**Comment by Prokyonn** [comment]
@MrBuchnev Are you sure that the domain in your webspace is correctly configured and equivalent with the domain in your url in the browser?

Can you try it out with checking the toggler for `All domains`?

edit: If it still doesn't work, maybe you can upload your testing-repository (with the database dump) to github, then it will be easier for us to trace the error.

**Comment by niklasnatter** [comment]
@MrBuchnev Can you provide a repository that allows us to reproduce this problem?

**Comment by niklasnatter** [comment]
Closing this because of inactivity. Please feel free to create a new issue with a reproducible repository if the problem still occurs for you.
